ruby/neo4j/driver/internal/scheme.rb

```diff
@@ -5,28 +5,29 @@ module Neo4j::Driver::Internal::Scheme
   NEO4J_URI_SCHEME = 'neo4j'
   NEO4J_HIGH_TRUST_URI_SCHEME = 'neo4j+s'
   NEO4J_LOW_TRUST_URI_SCHEME = 'neo4j+ssc'
-
-  def validate_scheme!(scheme)
-    unless [BOLT_URI_SCHEME, BOLT_LOW_TRUST_URI_SCHEME, BOLT_HIGH_TRUST_URI_SCHEME, NEO4J_URI_SCHEME,
-            NEO4J_LOW_TRUST_URI_SCHEME, NEO4J_HIGH_TRUST_URI_SCHEME].include?(scheme)
-      raise ArgumentError, scheme ? "Invalid address format #{scheme}" : 'Scheme must not be null'
+  class << self
```

Comment:
The Schema class in java has only static methods, so it can be represented in ruby by a module as no instances are created. Did you have difficulties calling the `validate_scheme!` method the way it was set up?

Reply:
The reason I made it module level method is because I don't want to include `Scheme` in `DriverFactory` class.

Out of all code in `Scheme` module (constants and bunch of methods) only a single method was being used there. also  in `GraphDatabase` class another single method of `Scheme` was being used. I think it is better if we keep `Scheme` as a standalone module, and use it directly instead of calling it. Let me know if you think ottherwise.